label_studio/core/settings/base.py

```diff
@@ -486,3 +486,5 @@ def collect_versions_dummy(**kwargs):
 # default value for feature flags (if not overrided by environment or client)
 FEATURE_FLAGS_DEFAULT_VALUE = False
 
+# Strip harmful content from SVG files by default
+SVG_SECURITY_CLEANUP = True
```

Comment:
SVG_SECURITY_CLEANUP = get_bool_env('SVG_SECURITY_CLEANUP', True)

Reply:
our users should be able to do this without code modifications, we often ask them to fix things like this through the console

Reply:
> (if not overrided by environment or client)

The existing comment above makes how one should define/set these misleading. Why would it differ from how we set ```FEATURE_FLAGS_DEFAULT_VALUE```

also, i see we override settings with a conf file. i.e. ```core/settings/base.py```, i thought many uses would have their own.

i'll make the change you requested, just letting you know how i arrived at the decision.

Reply:
FEATURE_FLAGS_DEFAULT_VALUE should be `FEATURE_FLAGS_DEFAULT_VALUE = get_env('FEATURE_FLAGS_DEFAULT_VALUE', True)` too, we missed it. 

> i thought many usesr would have their own.
pip users won't have it :-)